activesupport/lib/active_support/configurable.rb

```diff
@@ -27,6 +27,19 @@ def #{key}; _get(#{key.inspect}); end
     end
 
     module ClassMethods
+      # Reads and writes attributes from a configuration OrderedOptions.
+      #
+      #   require "active_support/configurable"
```

Comment:
I don't think we typically call the require in examples (I could not find another example).
When using Rails this `require` isn't needed as all of ActiveSupport is already loaded?

Reply:
@p8 that's a good point. I found it odd as well, but drew inspiration directly from the `#config` method documentation found elsewhere in the module (which pre-dates this set of changes):

https://github.com/rails/rails/blob/a01b093469a30e1abcecf1a89c38b85ba7ddad10/activesupport/lib/active_support/configurable.rb#L168-L182